addons/addon-base-raas/packages/base-raas-services/lib/alb/alb-service.js

```diff
@@ -326,6 +327,56 @@ class ALBService extends Service {
     return `${prefix}-${id}.${domainName}`;
   }
 
+  /**
+   * Method to get the EC2 SDK client for the target aws account
+   *
+   * @param requestContext
+   * @param resolvedVars
+   * @param vpcId
+   * @returns {Promise<string>}
+   */
+  async findSubnet2(requestContext, resolvedVars, vpcId) {
```

Comment:
Didn't see a unit test for this method.

Reply:
We will add the test case and raise another PR